src/controllers/controller.scatter.js

```diff
@@ -37,6 +37,10 @@ defaults._set('scatter', {
 module.exports = function(Chart) {
 
 	// Scatter charts use line controllers
-	Chart.controllers.scatter = Chart.controllers.line;
+	Chart.controllers.scatter = Chart.controllers.line.extend({
```

Comment:
What's the reason for overriding and returning false here? Shouldn't that have already happened?

Reply:
From what i understand, with mixed chart, the default style applied is the style of the 'main' chart.
And for now, only `bar` or `line` are valid main types.
So the option `showLines` is never set to `false`. You'd have to do it manually in the child chart.
You can try commenting the line n°8187 and n°19040 in here https://codepen.io/anon/pen/zpWMOj
And you can check the console for `Chart.config.data.datasets[0].showLine`.

Reply:
But now that I think about it, is being able to show lines in a scatter chart a wanted feature ?  
If so, then this overriding would break it.

Reply:
I think lines on a scatter chart is something that should be possible and its currently the default behaviour

Reply:
So does that mean a test should be added via another PR to make sure it stays that way in the future ?

Reply:
I would think adding a test to ensure that a scatter chart can optionally use a line or not is a good idea. @simonbrunel thoughts?

Reply:
I don't see why scatter charts couldn't display lines, so yes these `lineEnabled` changes should be reverted. We could add a test indeed, but in a different PR.